### DISCUSSION TITLE
Build region

### DISCUSSION THREAD

**Discussion by ohyoxo** [vercel/community#5939] [Help] [answered]
### Summary

Can the default Build region be changed?

### Steps to Reproduce

Is it not supported to make changes?

**Comment by swarnava** [comment, marked as answer]
This isn't possible at the moment and the only region you're able to change is the Serverless Function region and Edge Function regions. All build traffic will come from a US based regions and this can't be changed.

If you need a build to happen in a specific region, we'd suggest raising this as a feature request to allow our engineering team to better understand the need/public interest for this feature: https://vercel.com/guides/where-to-submit-feature-requests

Please let us know if you have any further questions or issues.
